plugins/modules/aws_secret.py

```diff
@@ -166,8 +172,18 @@ def create_args(self):
         }
         if self.description:
             args["Description"] = self.description
+        if self.replica_regions:
+            args["AddReplicaRegions"] = self.replica_regions
         if self.kms_key_id:
             args["KmsKeyId"] = self.kms_key_id
+        if self.replica_regions:
+            add_replica_regions = []
+            for region in self.replica_regions:
+                if self.kms_key_id:
+                    add_replica_regions.append({'Region': region, 'KmsKeyID': self.kms_key_id})
+                else:
+                    add_replica_regions.append({'Region': region})
+            args["AddReplicaRegions"] = add_replica_regions
```

Comment:
Unfortunately this means that a newer version of botocore (v1.23.21) is needed to pass the CI integration tests.

```Traceback (most recent call last):
  File "/tmp/ansible_aws_secret_payload_yr96uaq4/ansible_aws_secret_payload.zip/ansible_collections/community/aws/plugins/modules/aws_secret.py", line 236, in create_secret
  File "/home/zuul/venv/lib/python3.8/site-packages/botocore/client.py", line 357, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/home/zuul/venv/lib/python3.8/site-packages/botocore/client.py", line 648, in _make_api_call
    request_dict = self._convert_to_request_dict(
  File "/home/zuul/venv/lib/python3.8/site-packages/botocore/client.py", line 696, in _convert_to_request_dict
    request_dict = self._serializer.serialize_to_request(
  File "/home/zuul/venv/lib/python3.8/site-packages/botocore/validate.py", line 297, in serialize_to_request
    raise ParamValidationError(report=report.generate_report())
botocore.exceptions.ParamValidationError: Parameter validation failed:
Unknown parameter in input: "AddReplicaRegions", must be one of: Name, ClientRequestToken, Description, KmsKeyId, SecretBinary, SecretString, Tags
fatal: [testhost]: FAILED! => {
    "boto3_version": "1.16.0",
    "botocore_version": "1.19.0",
```

Reply:
That's okey. But you should notices this in the documentation of the new parameter.